src/main/java/lotto/LottoWinningInfo.java

```diff
@@ -0,0 +1,46 @@
+package lotto;
+
+public enum LottoWinningInfo {
+    THIRD(3, 5000),
+    FOURTH(4, 50000),
+    FIFTH(5, 1500000),
+    SIXTH(6, 2000000000);
```

Comment:
언더 스코어 표기법을 활용하면 가독성이 좋아져요 :)
```suggestion
    SIXTH(6, 2_000_000_000);
```

Reply:
안녕하세요! 말씀하신 부분 수정하였습니다 실행 할 수 있는 클래스를 누락했는데 그부분도 추가하여 올렸습니다! 좋은 리뷰 감사합니다